### ISSUE TITLE
How to start your blockchain platform?

### ISSUE THREAD

**Issue by bulldosers** [AplaProject/go-apla#131]
When I use the command `go get github.com/AplaProject/go-apla`, I just get some log such as:
```
# github.com/AplaProject/go-apla/packages/notificator
gopath/src/github.com/AplaProject/go-apla/packages/notificator/notificator.go:23: undefined: sync.Map
# github.com/AplaProject/go-apla/packages/autoupdate
gopath/src/github.com/AplaProject/go-apla/packages/autoupdate/autoupdate.go:50: undefined: os.Executable
```
I don't know how to start your blockchain network just like [geth](https://geth.ethereum.org/) client.
Look forward your reply and help, thanks.

**Comment by rpoletaev** [comment]
go version?

**Comment by bulldosers** [comment]
yeah, i just want to run it like geth that i am more familar with and
exactly said,  run a private chain network locally. thx.

**Comment by pavelfess** [comment]
@bulldosers please specify your Golang version for more detailed feedback.

**Comment by bulldosers** [comment]
Thanks for your replay and I upgrade my go version to 1.9.3. When I run command line `~/apla/go-apla
`, I get the log 
```
INFO[0000] Config file missing.                         
INFO[0000] started with                                  version=0.1.6b11 work_dir=/home/fuwanzeng/apla
```
and what I need to do to continue ? Is there some web-API that could be used to interact with the blockchain platform just like [geth-client](https://github.com/ethereum/wiki/wiki/JSON-RPC)? Or is there some wrong understanding disunderstand with the feature of the blockchain platform?

**Comment by c-darwin** [comment]
Hello @bulldosers,
Sorry for delay.

**Short instruction**

1. Download https://github.com/centrifugal/centrifugo/releases

2. config.json
```JSON
{
  "secret": "yWaC39F9zWgP4xqp",
  "web": true,
  "admin_password": "password",
  "admin_secret": "test",
  "namespaces": [
    {
      "name": "public",
      "anonymous": true,
      "publish": true,
      "watch": true,
      "presence": true,
      "join_leave": true,
      "history_size": 10,
      "history_lifetime": 30,
      "recover": true
    }
  ]
}
```
3. ./centrifugo

4. Create database "testing" 
5. Clone and go build https://github.com/AplaProject/go-apla/tree/release 
6. ./go-apla -dbUser=postgres -dbName=testing -initDatabase=1 -centrifugoSecret=yWaC39F9zWgP4xqp -centrifugoUrl=http://localhost:8000/  -generateFirstBlock=1 -workDir=/YOUR_DIR/ -configPath=/dev/null
7. Setup https://github.com/AplaProject/apla-front
8. REACT_APP_API_URL='http://localhost:7079/api/v2' yarn build
9. yarn start
10. http://localhost:3000 -> import existing key -> /YOUR_DIR/PrivateKey
12. Admin tools->import https://raw.githubusercontent.com/GenesisKernel/apps/master/demo_page.json
12. Goto home page
